roles/sat6repro/vars/main.yml

```diff
@@ -18,11 +18,17 @@ config_files_tar_file_name: config_files.tar.gz
 pgsql_data_tar_file_name: pgsql_data.tar.gz
 mongo_data_tar_file_name: mongo_data.tar.gz
 
+# Restoring a pulp_data.tar file from a backup is optional. Change include_pulp_data to false to skip.
+# If you skip this step, repositories will still be set up, but will be unsynced.
+# Any manually uploaded content will have to be re-uploaded.
+include_pulp_data: false
+pulp_data_tar_file_name: pulp_data.tar
+
 
 # Registration info - Credentials to register your host to RH portal
-activationkey: ak1
+activationkey: changeme
 org: 111111
-
+pool: 111111
```

Comment:
I've been using it for an activation key, but probably unnecessarily...  I can remove it.